src/Filters/response.jl

```diff
@@ -128,3 +154,13 @@ end
 function hz_to_radians_per_second(hz, fs)
     hz * ((2 * pi) / fs)
 end
+
+function _is_sym(x::AbstractArray)
+    n = length(x) ÷ 2
+    return all([x[1+i] == x[end-i] for i in 0:n])
+end
+
+function _is_anti_sym(x::AbstractArray)
+    n = length(x) ÷ 2
+    return all([x[1+i] == -x[end-i] for i in 0:n])
```

Comment:
```suggestion
    return all(x[1+i] == -x[end-i] for i in 0:n)
```
Here, too, we can optimize by avoiding the temporary vector. We have to go to `n` though, to ensure the middle element for odd length is zero (by comparing with itself negated). Or we use `n = (length(x) + 1) ÷ 2` and stop at `n-1`; not sure if worth it.